src/index.ts

```diff
@@ -38,15 +41,31 @@ function init({
 		return;
 	}
 
-	if (typeof isInUsa === 'undefined') {
+	if (typeof country === 'undefined') {
 		throw new Error(
-			'CMP initialised without `isInUsa` property. `isInUsa` is required.',
+			'CMP initialised without `country` property. A 2-letter, ISO ISO_3166-1 country code is required.',
 		);
 	}
 
+	const framework = getFramework(country);
+
 	window.guCmpHotFix.initialised = true;
 
-	CMP = isInUsa ? CCPA : TCFv2;
+	switch (framework) {
+		case 'tcfv2':
+		default:
+			CMP = TCFv2;
+			break;
+		case 'ccpa':
+			CMP = CCPA;
+			break;
+		case 'aus':
+			CMP = AUS;
+			break;
```

Comment:
not a big deal, it just took me a while to work out what was happening here
```suggestion
		case 'ccpa':
			CMP = CCPA;
			break;
		case 'aus':
			CMP = AUS;
			break;
		case 'tcfv2':
		default: // default is also 'tcfv2'
			CMP = TCFv2;
			break;
```

Reply:
I checked and `default:` does not have to be at the bottom. If the convention makes it easier to parse, happy to apply your suggestion.

Reply:
yeah i htink it will work just fine, i was just a bit lost trying to work it out, that's all. not a blocker :)